source/views/streaming/streams/list.tsx

```diff
@@ -23,52 +23,32 @@ const styles = StyleSheet.create({
 	},
 })
 
-export const StreamListView = (): JSX.Element => {
-	let [error, setError] = React.useState<Error | null>(null)
-	let [loading, setLoading] = React.useState(true)
-	let [refreshing, setRefreshing] = React.useState(false)
-	let [streams, setStreams] = React.useState<
-		Array<{title: string; data: StreamType[]}>
-	>([])
-
-	React.useEffect(() => {
-		try {
-			getStreams().then(() => {
-				setLoading(false)
-			})
-		} catch (error) {
-			if (error instanceof Error) {
-				setError(error)
-			} else {
-				setError(new Error('unknown error - not an Error'))
-			}
-			return
-		}
-	}, [])
-
-	let refresh = async (): Promise<void> => {
-		setRefreshing(true)
-		await getStreams(true)
-		setRefreshing(false)
-	}
+const groupStreams = (entries: StreamType[]) => {
+	let grouped = groupBy(entries, (j) => j.$groupBy)
+	return toPairs(grouped).map(([title, data]) => ({title, data}))
+}
 
-	let getStreams = async (
-		reload?: boolean,
-		date: Moment = moment.tz(timezone()),
-	) => {
-		let dateFrom = date.format('YYYY-MM-DD')
-		let dateTo = date.clone().add(2, 'month').format('YYYY-MM-DD')
+const useStreams = (date: Moment = moment.tz(timezone())) => {
```

Comment:
Any particular reason to prefer a `const` `=>` function here?

Reply:
As opposed to `function useStreams(date: Moment = moment.tz(timezone())) {`? Not that I can think of.

The main reason to default to arrow functions is that they behave more like you'd expect wrt. `this` (they inherit `this` from the caller's context, instead of the defintion location, IIRC). So there are probably fewer pitfalls if we do everything as `const/let =>`…